setup.py

```
@@ -94,21 +94,13 @@ def _pythonpath_env(extra_path):
     try:
         from distutils.core import find_packages
     except ImportError:
-        raise ImportError(
-            "setuptools is required for building phaseshifts. Please install setuptools."
-        )
+        raise ImportError("setuptools is required for building phaseshifts. Please install setuptools.")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)